scripts/dovecot/build-obs.sh

```diff
@@ -1,13 +1,19 @@
 #!/bin/bash
 
+# this script requires wget, osc and debuild installed.
+
+# Define path of your local OBS repository
+OBS_PATH=~/obs/home:deltachat/dovecot/
```

Comment:
a strange default -- what about just `obs-dovecot` as subdir of current dir?   See also the modified invocation suggestion above.

Reply:
it's a convention from internal suse stuff. If you check out your entire home-repo, as recommended, you get all your packages into this folder. 

But you're right, in that case it makes more sense here.

Reply:
done